GrassChoppers/Modules/Home/HomeDataManager.swift

```diff
@@ -1,13 +1,24 @@
 protocol HomeDataManaging: class {
     var delegate: HomeDataManagingDelegate? { get set }
+    
+    func requestInitialState()
 }
 
 protocol HomeDataManagingDelegate: class {
-    
+    func didFetch(name: String)
```

Comment:
This protocol defines what it means to be a delegate of the `HomeDataManager`. As it is a protocol, its only job is to declare what functions & properties the underlying class must have. If a class wants to conform to this protocol (like in the case of a delegate), then it must have this function.